packages/bpk-component-scrollable-calendar/src/_variables.scss

```diff
@@ -0,0 +1,25 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins/index';
+
+$shadow-bottom: 0 $bpk-one-pixel-rem 0 0 $bpk-color-gray-100;
+$shadow-left: -$bpk-one-pixel-rem 0 0 0 $bpk-color-gray-100;
+$shadow-right: $bpk-one-pixel-rem 0 0 0 $bpk-color-gray-100;
+$calendar-width: 7 * ($bpk-calendar-day-size + $bpk-calendar-day-spacing);
+$calendar-height: 7 * ($bpk-calendar-day-size + $bpk-calendar-day-spacing);
```

Comment:
`$calendar-height` is the only one of these that's in use - do the others need to be here?